editor/filesystem_dock.cpp

```diff
@@ -1064,6 +1064,13 @@ void FileSystemDock::_folder_option(int p_option) {
 				child = child->get_next();
 			}
 			break;
+		case FOLDER_SHOW_IN_EXPLORER:
+			String path = item->get_metadata(tree->get_selected_column());
+			String dir = GlobalConfig::get_singleton()->globalize_path(path);
+			dir = dir.substr(0, dir.find_last("/"));
```

Comment:
Well, if I right click on a folder and use "show in file browser", I would expect it to open the folder itself, otherwise, I have to make one more click. Anyway, that's open to discussion, and also nonblocking.